ni/measurements/data/v1/data_store_service.proto

```diff
@@ -235,6 +235,8 @@ message PublishMeasurementBatchRequest {
   oneof values {
     ni.protobuf.types.Vector scalar_values = 2;
   }
+  // Optional. Any notes to be associated with the captured measurement.
+  string notes = 3;
```

Comment:
I think elsewhere this is called "measurement_notes" we should be consistent.  I imagine based on the direction of name we should remove the prefix elsewhere.